### DISCUSSION TITLE
What is the meaning of the blue overlay icon and associated message?

### DISCUSSION THREAD

**Discussion by Eniki** [Devolutions/UniGetUI#2972] [Q&A] [answered]
I'm new to UniGetUI - looks very useful and I updated quite a few packages successfully!
But when I switchd to the "installed packages" vew I would a number of packages with a overlay icon (blue circle with an unidentifiable symbol in it) next to the package name. When I hover with the mouse over the name I get a pop up with text like "this package can be updated to Version x.y" - but the installed version in the Version column lists exactly the same version x.y. See attachment for an example.
So how should I read this icon and the associated message?
In the "Software update view" the same package is not listed - which is OK, as there is no later update available.

![Image 659x165](https://github.com/user-attachments/assets/78a757ed-9c0b-4941-9f58-82964972ea47)

Thanks in advance for any insights.

**Comment by marticliment** [comment, marked as answer]
This seems to be a bug, I will fix this

**Reply by Eniki** [reply]
Thanks for the quick assessment!
